cmd/broker/binding_test.go

```diff
@@ -116,3 +119,131 @@ func TestBinding(t *testing.T) {
 	})
 
 }
+
+func TestDeprovisioningWithExistingBindings(t *testing.T) {
+	// given
+	cfg := fixConfig()
+	// Disable EDP to have all steps successfully executed
+	cfg.EDP.Disabled = true
+	suite := NewBrokerSuiteTestWithConfig(t, cfg)
+	defer suite.TearDown()
+	iid := uuid.New().String()
+	bindingID1 := uuid.New().String()
+	bindingID2 := uuid.New().String()
+
+	response := suite.CallAPI("PUT", fmt.Sprintf("oauth/v2/service_instances/%s?accepts_incomplete=true", iid),
```

Comment:
In our cases `accepts_incomplete` should be always false. We are supporting only synchronized invocations.

Reply:
This is copy paste of our code which calls provisioning. This call is served asyynchronously.